src/Element/UKAddressLookup.php

```diff
@@ -59,6 +59,9 @@ public static function getCompositeElements(array $element) {
     $element_list['address_1']['#prefix'] = '<div class="js-address-entry-container">';
     $element_list['postcode']['#suffix'] = '</div>';
 
+    // Custom error message for address line 1.
+    $element_list['address_1']['#required_error'] = t('You must enter an address.');
```

Comment:
Not blocking this. But might be worth moving to StringTranslationTrait at some point.

Reply:
Yeah.  It caused me some unease as well.  But eventually stuck to the t() function for consistency with the rest of the file.  I have just raised a [ticket](https://github.com/localgovdrupal/localgov_forms/issues/99) to replace the entire lot.